src/arvdevice.c

```diff
@@ -393,6 +393,35 @@ _get_feature (ArvDevice *device, GType node_type, const char *feature, GError **
 	return node;
 }
 
+/**
+ * arv_device_get_representation:
+ * @device: a #ArvDevice
+ * @feature: feature name
+ * @error: a #GError placeholder, %NULL to ignore
+ *
+ * Return: enum ArvGcRepresentation, -1 if not available.
+ *
+ * Since: 
+ */
+
+ArvGcRepresentation
```

Comment:
Remove the error placeholder

Reply:
updated in b9c1305b92d0322ec1c7f1ebef44a47d78db6a99